data/channels.csv

```diff
@@ -21239,31 +21239,31 @@ NLTV55.us,NLTV 55,,,,US,US-HI,Hilo,s/US-HI,eng,general,FALSE,,,,https://naleo.tv
 NMBTV.us,NMBTV,,,,US,US-FL,North Miami Beach,s/US-FL,eng,legislative,FALSE,,,,https://www.citynmb.com/174/NMBTV-Live-Stream,https://i.imgur.com/HTNoNnh.png
 NMTV.us,NMTV,North Metro TV,,,US,US-MN,Blaine,s/US-MN,eng,general,FALSE,,,,https://northmetrotv.com/channel-15-live-stream/,https://i.imgur.com/1JiGTMN.png
 NNPSTV.us,NNPS-TV,,,,US,US-VA,Newport News,s/US-VA,eng,education,FALSE,,,,https://nnpstv.com/,https://i.imgur.com/OXVZo7u.png
-NNT.ru,NNT,ННТ,,,RU,,,c/RU,rus,,FALSE,,,,https://nnttv.ru/,https://i.imgur.com/Q6CdX8Z.png
+NNT.ru,NNT,ННТ,,,RU,,Makhachkala,c/RU,rus,general,FALSE,,,,https://nnttv.ru/,https://i.imgur.com/Q6CdX8Z.png
 NNTV.ru,NNTV,ННТВ,,,RU,,Nizhny Novgorod,c/RU,rus,news,FALSE,,,,https://nntv.tv/,https://i.imgur.com/J3r1dqY.png
 NNTV.us,NNTV,,,,US,US-VA,Newport News,s/US-VA,eng,general,FALSE,,,,http://newportnewsva.cvp.swagit.com/live,https://i.imgur.com/IRqyAcA.png
 NNTV2.us,NNTV2,,,,US,US-VA,Newport News,s/US-VA,eng,general,FALSE,,,,http://newportnewsva.cvp.swagit.com/live/?channel=nntv2,https://i.imgur.com/IRqyAcA.png
 NNVeracruz.mx,NNVeracruz,,,,MX,MX-VER,Veracruz,c/MX,spa,news,FALSE,,,,https://www.nnveracruz.com.mx/,https://i.imgur.com/7H3MtwG.png
 noa4Hamburg.de,noa4 Hamburg,,,,DE,,Hamburg,c/DE,deu,general,FALSE,,,,https://www.noa4.de/hamburg.html,https://i.imgur.com/7A2LRKI.png
 noa4Norderstedt.de,noa4 Norderstedt,,,,DE,,Norderstedt,c/DE,deu,general,FALSE,,,,https://www.noa4.de/norderstedt.html,https://i.imgur.com/941nrGb.png
-NobleTV.eg,Noble TV,,,,EG,,Dnipro,c/EG,ara,,FALSE,,,,https://noble.tv/,https://i.imgur.com/yjAhb4w.png
+NobelTV.ua,Nobel TV,,,,UA,,Dnipro,c/UA,ukr,general,FALSE,2003-08-07,2023-06-05,,https://noble.tv/,https://i.imgur.com/yjAhb4w.png

```

Comment:
Where did you find information that this channel is broadcast from Ukraine? The official website does not even mention the Ukrainian language:

![image](https://github.com/user-attachments/assets/133eea34-1f02-4b90-9a54-57511ad1a513)

https://noble.tv/